src/std/make.ss

```diff
@@ -20,6 +25,64 @@
         ldflags
         cppflags)
 
+;; TODO: move to a new :std/misc/hash that includes most/all of :clan/utils/hash ?
+(def none# '(none#))
```

Comment:
I would avoid using a `#` in the name, that's the namespace separator.

Reply:
Fixed.